src/components/common/Actions.vue

```diff
@@ -5,39 +5,35 @@
       <v-btn id="save-btn" large
         :disabled="!isEntityType || isBusySaving"
         :loading="stateModel.isSaving"
-        @click="onClickSave()"
-      >
+        @click="onClickSave()">
         <span>Save</span>
       </v-btn>
 
       <v-btn id="save-resume-btn" large
         :disabled="!isEntityType || isBusySaving"
         :loading="stateModel.isSavingResuming"
-        @click="onClickSaveResume()"
-      >
+        @click="onClickSaveResume()">
         <span>Save and Resume Later</span>
       </v-btn>
     </div>
 
     <div class="buttons-right">
       <v-fade-transition hide-on-leave>
         <v-btn id="back-btn" large outlined
-          to="/define-company"
+          v-bind:to="previousRoute"
```

Comment:
Not sure but i don't believe `v-bind` is required if your prefer the shorthand `:`. Up to you!

Reply:
Done